src/components/DropDown.tsx

```diff
@@ -0,0 +1,17 @@
+import '../styles/DropDown.scss';
+
+const DropDown = ({ callback }: { callback: Function }) => {
```

Comment:
Dropdown is too generic name, especially if you use it only for showing sorting options.